.circleci/config.yml

```diff
@@ -28,9 +28,34 @@ jobs:
             if [[ $CIRCLE_BRANCH == "master" ]]; then
               docker push chaspy/ask:latest
             fi
+  deploy:
+    docker:
+      - image: circleci/python:3.6.1
+    environment:
+      AWS_DEFAULT_OUTPUT: json
+    steps:
+      - checkout
+      - run:
+          name: Install the AWS CLI
+          command: |
+            sudo pip install -r requirements.txt
+      - save_cache:
+          key: v1-{{ checksum "requirements.txt" }}
+          paths:
+            - "venv"
+      - run: |
+          curl https://raw.githubusercontent.com/silinternational/ecs-deploy/master/ecs-deploy | sudo tee /usr/local/bin/ecs-deploy
+          sudo chmod +x /usr/local/bin/ecs-deploy
+      - run: ecs-deploy -n ask-service -k $AWS_ACCESS_KEY_ID -s $AWS_SECRET_ACCESS_KEY -r $AWS_DEFAULT_REGION -c ask-cluster -i chaspy/ask:latest
```

Comment:
I've added the environment variables to CircleCI
https://circleci.com/gh/chaspy/ask/edit#env-vars
<img width="1286" alt="screenshot 2019-02-09 22 33 19" src="https://user-images.githubusercontent.com/10370988/52522002-bf81ba00-2cba-11e9-881f-f41ea551499d.png">